bozidarsevo/WordPress-iOS
# [WiP] Discover posts sorting control

Ref [#14765](https://github.com/wordpress-mobile/WordPress-iOS/issues/14765)
WPKit PR: https://github.com/wordpress-mobile/WordPressKit-iOS/pull/340

To test:

Open the Reader tab and check if the control is showing properly.

Things left to check:
- [ ] if used strings are localized
- [ ] use proper icons
- [ ] how should the control behave when touched/highlighted
- [ ] if accessibility is set correctly

![Simulator Screen Shot - iPhone 12 Pro - 2021-01-31 at 22 48 55](https://user-images.githubusercontent.com/6242034/106399112-01abe500-6417-11eb-9c6b-d664f2fa0e95.png)

PR submission checklist:
